text/0000-add-state-field.md

```diff
@@ -0,0 +1,85 @@
+# Meta
+[meta]: #meta
+- Name: Add State to RFC Metadata
+- Start Date: 2021-07-14
+- Author(s): @aemengo, @ekcasey
+- State: Approved | On Hold
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This is a proposal to add a `State:` field to the `Meta` section of the RFC template.
+
+# Definitions
+[definitions]: #definitions
+
+**Core team:** The party that votes on the approval of RFCs.
+
+# Motivation
+[motivation]: #motivation
+
+This RFC formally introduces the **"On Hold"** state for an RFC, in a way that's clear to the community.
```

Comment:
I think it's worth adding an explanation of why this field is needed.

Reply:
I'm partial to Stephen's thoughts here, saying:

> I suggest we require individual RFCs to specify why they might be in the On Hold state, and not define a reason for On Hold in this RFC.

https://github.com/buildpacks/rfcs/pull/177#discussion_r673301936

Reply:
> When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone).

Also, there's a postpone disposition today that we don't take advantage of. I wonder if that's something we can use here?

Reply:
@aemengo, my comment is on the motivation section:
```This RFC formally introduces the **"On Hold"** status for an RFC, in a way that's clear to the community.```
What do you mean by "in a way that's clear to the community"?
I think it's worth adding an explanation on why do we want this RFC.
Maybe giving an example or two may help?

Reply:
@yaelharel 

> What do you mean by "in a way that's clear to the community"?

I suppose I meant, in a way that is clear to anyone reading the RFC. As opposed to, through word-of-mouth.

> Maybe giving an example or two may help?

I don't think adding a `Status:` to RFCs is a novel idea. We can see them as part of other RFC templates, for instance [Tekton](https://github.com/tektoncd/community/tree/main/teps#teps).